### PR TITLE
TESTING fixes

### PR DESCRIPTION
**Pull request by eshpc**
**Description**

Some minor fixes in TESTING code:
1. Correct type mismatches for ALPHA and BETA in calls to xLASET.
2. Tidy some indentation issues.

### CODE REVIEW

File: TESTING/LIN/cdrvhe_aa_2stage.f [hunk 1]
```diff
@@ -450,7 +450,7 @@ SUBROUTINE CDRVHE_AA_2STAGE(
 c     $                                  IWORK, AINV, LDA, RWORK,
 c     $                                  RESULT( 2 ) )
 c                     NT = 2
-					  NT = 1
+                     NT = 1
```
Comment (h-vetinari) [inline TESTING/LIN/cdrvhe_aa_2stage.f:453]: This still seems to be mismatched...

**Review by langou — APPROVED** [review]

**Comment by langou** [comment]
Thanks @h-vetinari, looks good to me. Cheers @langou

File: TESTING/LIN/cdrvhe_aa_2stage.f (hunk 1 above)
Comment (weslleyspereira) [reply TESTING/LIN/cdrvhe_aa_2stage.f]: Actually, there was an extra space in the comment block. I changed it to avoid confusion.

**Review by weslleyspereira — APPROVED** [review]

**Comment by weslleyspereira** [comment]
Thanks @eshpc!

**Pull request merged**